sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/MergeRowsExec.scala

```diff
@@ -49,7 +48,21 @@ case class MergeRowsExec(
 
   override lazy val metrics: Map[String, SQLMetric] = Map(
     "numTargetRowsCopied" -> SQLMetrics.createMetric(sparkContext,
-      "Number of target rows copied unmodified because they did not match any action."))
+      "Number of target rows copied unmodified because they did not match any action"),
```

Comment:
Minor: It seems like we call it "action" here and "clause" in all other metrics. It would be nice to align.